aspnetcore/migration/31-to-50.md

```diff
@@ -81,7 +81,6 @@ For apps using Docker, use a base image that includes ASP.NET Core 5.0. For exam
 docker pull mcr.microsoft.com/dotnet/core/aspnet:5.0
 ```
 
-<!-- uncomment after the breaking changes have been published -->
-<!-- ## Review breaking changes
+## Review breaking changes
 
-Review 3.1-to-5.0 breaking changes across .NET Core, ASP.NET Core, and Entity Framework Core at [Breaking changes for migration from version 3.1 to 5.0](/dotnet/core/compatibility/3.1-5.0). -->
+For breaking changes from .NET Core 3.1 to .NET 5.0, see [Breaking changes for migration from version 3.1 to 5.0](/dotnet/core/compatibility/3.1-5.0). Breaking changes for ASP.NET Core and Entity Framework Core are also included in the list.
```

Comment:
This version is fine, but I'd be tempted to be less verbose. This is the breaking changes doc, the H2 is ` Review breaking changes`, do you really need to say that 4 times? The following is my suggestion but this version is fine.

Reply:
The challenge here is readers will also be interested in the changes to .NET itself. When going from 3.1 to 5.0, you shift from a product name of .NET Core to simply .NET.